module.flow.js

```diff
@@ -63,6 +65,11 @@ declare module '@solana/web3.js' {
     | SignatureSuccess
     | TransactionError;
 
+  declare export type SignatureStatus = {
+    slot: number,
+    status: SignatureSuccess | TransactionError,
```

Comment:
`SignatureStatusResult`?

Reply:
Will incorporate this here when I rebase: https://github.com/solana-labs/solana-web3.js/pull/742